examples/nodejs/token-vending-machine/lambda/token-vending-machine/handler.ts

```diff
@@ -34,13 +34,13 @@ export const handler = async (event: APIGatewayProxyEvent): Promise<APIGatewayPr
   }
 };
 
-interface VendedAuthToken {
-  authToken: string;
+interface VendedApiKey {
+  apiKey: string;
   expiresAt: number;
 }
 
-async function vendAuthToken(vendorAuthTokenSecretName: string, headers: APIGatewayProxyEventHeaders): Promise<VendedAuthToken> {
-  const momentoAuthClient = await getMomentoAuthClient(vendorAuthTokenSecretName);
+async function vendApiKey(vendorApiKeySecretName: string, headers: APIGatewayProxyEventHeaders): Promise<VendedApiKey> {
```

Comment:
should it be named `vendDisposableToken` instead if the TVM is using `generateDisposableToken`?

Reply:
nice catch, thanks!

Reply:
fixed in https://github.com/momentohq/client-sdk-javascript/pull/877/commits/a0d8beddd9a32923612fd466851b9c00198832d7